CMakeLists.txt

```diff
@@ -1,27 +1,31 @@
 #-----------------------------------------------------------------------------
-cmake_minimum_required(VERSION 3.5)
-foreach(p
-  ## Only policies introduced after the cmake_minimum_required
-  ## version need to explicitly be set to NEW.
-#Policies Introduced by CMake 3.11
-CMP0072  # FindOpenGL prefers GLVND by default when available.
-#Policies Introduced by CMake 3.10
-CMP0071  # Let AUTOMOC and AUTOUIC process GENERATED files.
-CMP0070  # Define file(GENERATE) behavior for relative paths.
-#Policies Introduced by CMake 3.9
-CMP0069  # INTERPROCEDURAL_OPTIMIZATION is enforced when enabled.
-CMP0068  # RPATH settings on macOS do not affect install_name.
-#Policies Introduced by CMake 3.8
-CMP0067  # Honor language standard in try_compile() source-file signature.
-#Policies Introduced by CMake 3.7
-CMP0066  # Honor per-config flags in try_compile() source-file signature.
-  )
-  if(POLICY ${p})
-    cmake_policy(SET ${p} NEW)
-  endif()
-endforeach()
+cmake_minimum_required(VERSION 3.9.5)
```

Comment:
Should we standardize on 3.13.4 like Slicer ?

Reply:
I am fine with the update to 3.13.4.  This PR is over a year old and I want to get it off my todo list.

Reply:
Done.